README.md

```diff
@@ -41,8 +41,10 @@ provides links to documentation, getting-started guides and release downloads.
 
 ## Visualization
 
-To visualize graphs stored in JanusGraph, you can use any of the following
-tools:
+JanusGraph has a web-based graph visualizer located in [janusgraph-visualizer](https://github.com/JanusGraph/janusgraph-visualizer) 
```

Comment:
I notice the below section still mentions `[Gremlin-Visualizer](https://github.com/prabushitha/gremlin-visualizer)`. Shall we somehow remove that and mention this is a fork of that? Or any other wording to avoid confusion.

Reply:
I just added a sentence to clarify that this was the original repository on which JanusGraph was built on.
I think, it makes sense to leave it in case gremlin-visualizer will decide to diverge in the future. In that case, users will be able to choose between visualizers they prefer more. 
That said, I doubt it will diverge, so I'm OK just removing it as well.